packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -585,6 +598,116 @@ class SingleActivator with Diagnosticable implements ShortcutActivator {
   }
 }
 
+/// A shortcut combination that is triggered by a key event of a character.
+///
+/// Keys often produce different characters when combined with modifiers. For
+/// example, it might be helpful for the user to bring up a help menu by
+/// pressing the question mark ('?'). However, there is no logical key that
+/// directly represents a question mark. Althouh 'Shift+Slash' produces a '?'
+/// character on a US keyboard, its logical key is still considered a Slash key,
+/// and hard-coding 'Shift+Slash' in this situation is unfriendly to other
+/// keyboard layouts.
+///
+/// The [CharacterActivator] is useful when the shortcut intent should be
+/// activated upon an event that produces a character undefined by a logical
```

Comment:
I agree that it's a bit confusing, so I'm taking it out. But what I meant was, if you want a `slash` character, you can just use the `slash` logical key, since practically they should function in the same way.